File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Chart.xUnit._VB/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.vb
'{[{
Imports Param_RootNamespace.Core.Services
'}]}

Public Class Tests
    '^^
    '{[{

    ' TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
    ' This test serves only as a demonstration of testing functionality in the Core library.
    <Fact>
    Public Async Sub EnsureSampleDataServiceReturnsChartDataAsync()
        Dim actual = Await SampleDataService.GetChartDataAsync()

        Assert.NotEmpty(actual)
    End Sub
    '}]}
End Class

File: templates/Uwp/_comp/_shared/Test.UnitTests.Core.SampleData.Chart.xUnit/Param_ProjectName.Core.Tests.xUnit/Tests_postaction.cs
//{[{
using Param_RootNamespace.Core.Services;
//}]}

namespace Param_RootNamespace.Core.Tests.XUnit
{
    public class Tests
    {
        //^^
        //{[{

        // TODO WTS: Remove or update this once your app is using real data and not the SampleDataService.
        // This test serves only as a demonstration of testing functionality in the Core library.
        [Fact]
        public async void EnsureSampleDataServiceReturnsChartDataAsync()
        {
            var actual = await SampleDataService.GetChartDataAsync();

            Assert.NotEmpty(actual);
        }
        //}]}
    }
}
